docs/configuration-reference/platforms/aws.md

```diff
@@ -117,6 +121,13 @@ cluster "aws" {
 
   enable_reporting = false
 
+  oidc {
+    issuer_url = var.oidc_issuer_url
```

Comment:
Please make the variables aligned, as `terraform fmt` would do. That improves readability, separating keys from values and makes HCL snippets consistent.